### PR TITLE
Add fileutils.

### LINKED ISSUE #121: Include fileutils
Latest [0.6.1](https://github.com/gildor478/ocaml-fileutils/releases/tag/v0.6.1) release of fileutils is once again compatible with Windows and uses dune for its build process.

It's already included in the [opam-windows](https://github.com/fdopen/opam-repository-mingw/blob/opam2/packages/fileutils/fileutils.0.6.1/opam) repository, so it would be nice to bring it to opam-cross-windows as well.

However, there's still a chain of missing dependencies (ounit etc.) that needs to be in place to include it. Should be it done in multiple separate pull requests or it's ok to combine multiple packages in one?

### PR DESCRIPTION
**Pull request by dmbaturin**
From my quick test, installing with `opam install fileutils-windows`, compiling a program like `let () = FileUtil.ls "." |> List.iter print_endline` with `ocamlfind -toolchain windows ocamlopt -package fileutils -linkpkg -o test ./test.ml`, and running `wine ./test.exe` works correctly.

Hope I haven't messed anything up, this is first time I make a cross-version of a package.

Fixes #121

### CODE REVIEW

File: packages/fileutils-windows/fileutils-windows.0.6.1/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
```
Comment (pirbo) [inline packages/fileutils-windows/fileutils-windows.0.6.1/opam]: The maintainer of the -windows package is you, not Sylvain. I've been told to put myself

File: packages/fileutils-windows/fileutils-windows.0.6.1/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+authors: [ "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ocaml-fileutils"
+license: "LGPL-2.1 with OCaml linking exception"
+dev-repo: "git+https://github.com/gildor478/ocaml-fileutils.git"
+bug-reports: "https://github.com/gildor478/ocaml-fileutils/issues"
+doc: "https://gildor478.github.io/ocaml-fileutils/"
+
+build: [
+  ["dune" "build" "-p" "fileutils" "-j" jobs "-x" "windows"]
+  ["dune" "runtest" "-p" "fileutils" "-j" jobs] {with-test}
```
Comment (pirbo) [inline packages/fileutils-windows/fileutils-windows.0.6.1/opam]: As the tests are not testing the windows version but the regular one, we usually remove the with-test targets in this repo (and the with-tests dependencies accordingly). 

(I've never had the case with the doc myself but I would say that the same applies and we leave to the regular package the duty to build the doc if the doc is wanted...)

File: packages/fileutils-windows/fileutils-windows.0.6.1/opam (hunk 1 above)
Comment (dmbaturin) [reply packages/fileutils-windows/fileutils-windows.0.6.1/opam]: Fair point. I'll fix this.

File: packages/fileutils-windows/fileutils-windows.0.6.1/opam (hunk 2 above)
Comment (dmbaturin) [reply packages/fileutils-windows/fileutils-windows.0.6.1/opam]: That's also a fair point.

**Comment by toots** [comment]
Thanks for the ping and for the review. Ready to merge this when the comments have been addressed.

**Pull request merged**